recipes/duckdb-cli/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "duckdb-cli" %}
+{% set version = "0.9.1" %}
+{% set extensions = "httpfs;json;parquet" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/duckdb/duckdb/archive/v{{ version }}.tar.gz
+  sha256: 37a43188d9354ce3ca101b2b118d867f5f76d04c3b83c09d86fd7508351a631b
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script:
+    - |
+      # echo "duckdb_extension_load(httpfs)" > extension/extension_config_local.cmake
+      make extension_configuration
+      DUCKDB_EXTENSIONS='{{ extensions }}' VCPKG_TOOLCHAIN_PATH="$VCPKG_ROOT/scripts/buildsystems/vcpkg.cmake" make
+      cp build/release/duckdb $PREFIX/bin
+      cp -R build/release/extension $PREFIX/
+  
+requirements:
+  host:
+    - cmake
+    - make
+    - vcpkg
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  run:
+    - libssh2
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+outputs:
+
+  - name: {{ name | lower }}
+    version: {{ version }}
+    files:
+      - bin/duckdb
+    requirements:
+      run:
+        - libssh2
+    test:
+      commands:
+        - which duckdb
+
+  - name: duckdb-httpfs
+    version: {{ version }}
+    files:
+      - extension/httpfs/httpfs.duckdb_extension
+    test:
+      commands:
+        - ls -lh $PREFIX/extension/httpfs/httpfs.duckdb_extension
```

This will at least return an error if the file does not exist.
```suggestion
        - test -f $PREFIX/extension/httpfs/httpfs.duckdb_extension
```

Is there a command we can run that shows which extensions are installed? That way we could do something like `duckdb --list-extensions | grep https` as the test.

Shouldn't the `duckdb-httpfs` extension depend on the `duckdb-cli` package?

---

Accepted